### PR TITLE
Updated smart contracts page -  with PSPs and Libraries

### PR DESCRIPTION
**Pull request by PierreOssun**
FYI Deleted the Ownable contract example as it was deprecated

### CODE REVIEW

File: docs/build/build-smart-contracts.md [hunk 1]
```diff
@@ -119,6 +119,16 @@ the state they consume but has since been deprecated.
 
 See the associated [pull request](https://github.com/paritytech/substrate/pull/9669) for more details.
 
+### Polkadot Standards Proposals (PSPs)
+
+Polkadot ecosystem got its own set of standards to fit ecosystem needs. Please visit [Polkadot Standards Proposals (PSPs) Github](https://github.com/w3f/PSPs).
```
Comment (salmad3) [inline docs/build/build-smart-contracts.md]:
```suggestion
Web3 Foundation supports proposals for Polkadot that define a set standards to fit ecosystem needs.
```

File: docs/build/build-smart-contracts.md [hunk 2]
```diff
@@ -119,6 +119,16 @@ the state they consume but has since been deprecated.
...
+
```
Comment (salmad3) [inline docs/build/build-smart-contracts.md]:
```suggestion
```

File: docs/build/build-smart-contracts.md [hunk 3]
```diff
@@ -119,6 +119,16 @@ the state they consume but has since been deprecated.
...
+These standards go through several acceptance phases, and the engagement of the whole community is needed to build valuable and future-proof standards.
```
Comment (salmad3) [inline docs/build/build-smart-contracts.md]:
```suggestion
These standards go through several acceptance phases, where the engagement of the whole community is needed to build valuable and future-proof standards.
```

File: docs/build/build-smart-contracts.md [hunk 4]
```diff
@@ -119,6 +119,16 @@ the state they consume but has since been deprecated.
...
+All the teams who will benefit from a standard needs to agree on its content.
+
+Some of these PSPs are targeting Substrate's `contracts` pallet:
+- [PSP22 - Fungible Token Standard](https://github.com/w3f/PSPs/blob/master/PSPs/psp-22.md)
+
```
Comment (salmad3) [inline docs/build/build-smart-contracts.md]:
```suggestion
Please visit [Polkadot Standards Proposals (PSPs) Github](https://github.com/w3f/PSPs) for more information.
```

File: docs/build/build-smart-contracts.md [hunk 5]
```diff
@@ -119,6 +119,16 @@ the state they consume but has since been deprecated.
 
 See the associated [pull request](https://github.com/paritytech/substrate/pull/9669) for more details.
 
+### Polkadot Standards Proposals (PSPs)
+
+Polkadot ecosystem got its own set of standards to fit ecosystem needs. Please visit [Polkadot Standards Proposals (PSPs) Github](https://github.com/w3f/PSPs).
+
+These standards go through several acceptance phases, and the engagement of the whole community is needed to build valuable and future-proof standards.
+All the teams who will benefit from a standard needs to agree on its content.
```
Comment (salmad3) [inline docs/build/build-smart-contracts.md]:
```suggestion
All the teams who will benefit from a standard need to agree on its content.
```

File: docs/build/build-smart-contracts.md [hunk 6]
```diff
@@ -136,14 +146,14 @@ parachains.
...
-#### Examples of Smart Contracts in `ink!`
+#### Libraries for Smart Contracts in `ink!`
 
 Collected below are some community examples of smart contracts in `ink!`. **Are you working on a
 smart contract example? Ask us to add it to this page!**
 
-- [Ownable](https://github.com/JesseAbram/foRust/): Port of the OpenZeppelin `Ownable` contract.
 - [OpenBrush](https://docs.openbrush.io/): an `ink!` library providing standard contracts based on
   [PSP](https://github.com/w3f/PSPs) with useful contracts and macros for building.
+- [Metis](https://github.com/patractlabs/metis): an Wasm contract standard library developed by Patract Labs
```
Comment (salmad3) [inline docs/build/build-smart-contracts.md]:
```suggestion
- [Metis](https://github.com/patractlabs/metis): a Wasm contract standard library, developed by [Patract Labs](https://patract.io/).
```

**Review by salmad3 — CHANGES_REQUESTED** [review]
Thanks @PierreOssun, left some suggestions

**Comment by PierreOssun** [comment]
Thanks @DannyS03 for the suggestions

**Review by salmad3 — APPROVED** [review]

**Pull request merged**